source/index.txt

```diff
@@ -40,6 +40,59 @@ deployments hosted in the following environments:
 
 .. include:: /includes/fact-environments.rst
 
+Related Tools and Libraries
```

Comment:
Yeah I wasn't really sure where to put it, but my thinking was that people don't usually read the landing page all the way through and most wouldn't see this section at all if it's at the bottom. But if you feel strongly that it shouldn't go at the top of the page, I can ask the PM (Alex B) if he's okay with moving it!

Reply:
for consistency with the other doc sets, i think it would be best toward the bottom of the list. can you ask alex (or i can)? you can also send him my way if there are more questions.

Reply:
Makes sense, I'll check with him